### ISSUE TITLE
Add the ability to custom navigation navigation and touch

### ISSUE THREAD

**Issue by nikrays** [usernein/tv-card#5]
Add the ability to custom navigation navigation and touch

**Comment by usernein** [comment]
Can you elaborate?

**Comment by nikrays** [comment]
> Можете ли вы уточнить?

I have a Phillips TV, I use the service as a navigation remote.
For example:

service:remote.send_command
 data:
   command: Up
 target:
   entity_id: remote.55pus7304_12_remote

 This way I can press up or down button respectively

**Comment by nikrays** [comment]
> Хммм я понял. Сначала я подумал, что вам нужны специальные кнопки для [использования этой карты с другими брендами, кроме Samsung](https://github.com/usernein/tv-card#notice) , но вы, возможно, уже используете это.
> 
> Я предполагаю, что ваш запрос касается тачпада, и он действительно работает только на samsung (раньше не замечал, спасибо), скоро исправлю
> 
> Для кнопок вы можете использовать пользовательские кнопки, упомянутые в ссылке выше:
> 
> ```yaml
> custom_keys:
>   up:
>     service: remote.send_command
>     service_data:
>       command: Up
>       entity_id: remote.55pus7304_12_remote
>   down:
>     service: remote.send_command
>     service_data:
>       command: Down
>       entity_id: remote.55pus7304_12_remote
> ```

Thanks for the work done, it's a pity that the flexibility of customization still leaves much to be desired, maybe it's better for me not to configure these buttons yet, but to wait for your release with a custom touchpad?

**Comment by usernein** [comment]
@nikrays yeah, you can wait for the touchpad support but you'll still need to customize all keys (not only the navigation-related ones) to call the service `remote.send_command` with the commands to your Philips TV

The only thing you can't use right now is the touchpad, but everything else (afaik) you can
But you still need to set custom buttons with the service calls

**Comment by nikrays** [comment]
> @nikraysда, вы можете дождаться поддержки тачпада, но вам все равно нужно будет настроить все клавиши (не только связанные с навигацией), чтобы вызывать службу `remote.send_command`с помощью команд для вашего телевизора Philips.
> 
> Единственное, что вы не можете использовать прямо сейчас, это тачпад, но все остальное (афаик) вы можете, но вам все равно нужно установить пользовательские кнопки со служебными вызовами.

Thank you, I understand, I started making these custom buttons a few days ago, there is of course an inconvenient nuance with icon mdi, it might be more convenient to do

**Comment by mattmansm** [comment]
Another vote from me to have the ability to remap the touchpad to use remote.send_command to control my Apple TV.

**Comment by Flo2410** [comment]
I also would love to be able to customize the touchpad. I would like to use it with the ```androidtv.adb_command``` service to control my chromecast.

**Comment by mattmansm** [comment]
Hi @usernein, any process with this? Thanks!

**Comment by ScottG489** [comment]
I'm interested in this as well!

As a pretty hacky workaround, you can listen for the events made from those buttons and act on them accordingly. The resulting events would have these distinct properties:

`event.domain: media_player`
`event.service: play_media`
`event.service_data.media_content_type: send_key`
`event.service_data.entity_id[0]: media_player.fake_entity_for_touchpad` (whatever media_player you have set on the card)
`payload.event.service_data.media_content_id: KEY_<whatever key you pressed>`

**Comment by RezzZ** [comment]
> <CUT>
> For buttons, you can use that custom buttons mentioned in the link above:
> 
> ```yaml
> custom_keys:
>   up:
>     service: remote.send_command
>     service_data:
>       command: Up
>       entity_id: remote.55pus7304_12_remote
>   down:
>     service: remote.send_command
>     service_data:
>       command: Down
>       entity_id: remote.55pus7304_12_remote
> ```

Hi! Im creating a remote for my nvidia shield. it also uses the remote.send_command.
Any chance you can add all button names to the example required for a complete functional dpad? And other buttons?
so we have:
up
down
left
right and select?
Also, when creating these custom keys, the icons disappear. What icons are used for up down left and right?

edit; think i already found it in the [tv-card]/tv-card.js file:
    "up": {"key": "KEY_UP", "icon": "mdi:chevron-up"},
    "left": {"key": "KEY_LEFT", "icon": "mdi:chevron-left"},
    "enter": {"key": "KEY_ENTER", "icon": "mdi:checkbox-blank-circle"},
    "right": {"key": "KEY_RIGHT", "icon": "mdi:chevron-right"},
    "down": {"key": "KEY_DOWN", "icon": "mdi:chevron-down"}
